### ISSUE TITLE
Allow to customize text color variables in insert/delete sections

### ISSUE THREAD

**Issue by Qqkyu** [otakustay/react-diff-view#187]
Right now, there is only the `--diff-text-color` variable, with which we can style text color. This is sometimes an issue when customizing background-colors of gutter and code sections.
We can override those styles, but it would be nicer if there were CSS classes for that:
```css
index.css

:root {
    --diff-background-color: initial;
    --diff-text-color: initial;
   ...
    --diff-gutter-insert-text-color: initial;
    --diff-gutter-delete-text-color: initial;
    ...
    --diff-code-insert-text-color: initial;
    --diff-code-delete-text-color: initial;
    ...
}

...

.diff-gutter-insert {
    background-color: var(--diff-gutter-insert-background-color);
    color: var(--diff-gutter-insert-text-color);
}

.diff-gutter-delete {
    background-color: var(--diff-gutter-delete-background-color);
    color: var(--diff-gutter-delete-text-color);
}

...

.diff-code-insert {
    background-color: var(--diff-code-insert-background-color);
    color: var(--diff-code-insert-text-color);
}

...

.diff-code-delete {
    background-color: var(--diff-code-delete-background-color);
    color: var(--diff-code-delete-text-color);
}

```

**Comment by otakustay** [comment]
Please review if https://github.com/otakustay/react-diff-view/pull/188 satisfies your customization requirements.

**Comment by Qqkyu** [comment]
LGTM, thanks for such a fast PR!

**Comment by otakustay** [comment]
Published version `2.6.0`
